src/Components/SmartComponents/SystemsPage/SystemsPage.js

```diff
@@ -25,14 +25,14 @@ import useDeepCompareEffect from 'use-deep-compare-effect';
 import * as APIHelper from '../../../Helpers/APIHelper';
 
 const createColumns = (defaultColumns, hasPatchAccess) => {
-    let [nameColumn, restColumns] = SYSTEMS_HEADER;
+    let [nameColumn, ...restColumns] = SYSTEMS_HEADER;
     let lastSeenColumn = defaultColumns.filter(({ key }) => key === 'updated');
```

Comment:
can we avoid the double loop over the `defaultColumns` array?

Reply:
I replaced `columns.filter(...)[0]` with `columns.find(...)` to be a bit faster and readable.